metals/src/main/scala/scala/meta/internal/metals/StatusBar.scala

```diff
@@ -70,7 +69,7 @@ final class StatusBar(
   }
 
   def trackSlowFuture[T](message: String, thunk: Future[T]): Unit = {
-    if (clientConfig.statusBarState == StatusBarState.Off)
+    if (!clientConfig.slowTaskIsOn)
```

Comment:
This seems to have been a wrong condition, since here we actually use the status bar and slow task in the else branch.